src/utils/subscription_dsl.py

```diff
@@ -66,7 +66,7 @@ def metadata(value):
 
 
 def throttling(value):
-    return node('throttling', value if value else 1)
+    return node('throttling', value if value is not None else 1)
```

Comment:
do we need some validation here? e.g. what if the client passes in: `"12", "12.3", 12.3, "nonsense"`. I don't think we've done validation in other places though, so it's probably best to leave it out for now and run a mammoth validation task sometime in the future for all input needing validation?

Reply:
When adding the test I thought the same. Not sure yet if that will bring more value than headaches (for this example of the subscription at least). Probably better to be decided with a mammoth task as you mention, let's discuss it later.